src/adapter/src/coord/timestamp_selection.rs

```diff
@@ -597,6 +598,8 @@ impl<T: fmt::Display + fmt::Debug + DisplayableInTimeline + TimestampManipulatio
     for TimestampExplanation<T>
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let now = SYSTEM_TIME();
+        writeln!(f, "                   system clock: {}", now)?;
```

Comment:
I'd like this to be a field on TimestampExplanation instead so we can use the session's pcx.wall_time field which has the time when the first statement of the txn was executed.